Makefile

```diff
@@ -49,5 +49,20 @@ locale:
 			msginit --no-translator -l $(LOCALE) -i $$pot -o locale/$(LOCALE)/LC_MESSAGES/`basename -s .pot $$pot`.po ; \
 		done
 
+localetest:
+	dennis-cmd lint --errorsonly locale/
+
+localeextract:
+	python manage.py extract
+	python manage.py merge
+
+localecompile:
+	cd locale; ./compile-mo.sh . ; cd --
+
+localerefresh: localeextract localetest localecompile compilejsi18n collectstatic
```

Comment:
This suggests that you're compiling to `.mo` files and then checking those in. Typically we compile `.po` -> `.mo` files during deploy.

How does this work in MDN?